src/app/components/category-display/category-display.component.html

```diff
@@ -0,0 +1,14 @@
+  <div class="container">
+    <div class="grid pt-5">
+        <div class="row">
+          @for (store of stores; track store.id) {
```

Comment:
In the view above, you may need to add CSS styling when stores are displayed in the selected category.
Use as an example the styling from our latest develop branch related to  "most famous store" (in the carousel component).
![category_stores_fix](https://github.com/perse-foni/WL_Delivery_Service/assets/4652993/ed90ec90-d3dd-477d-98bc-97b478929971)

Reply:
it is recommended to appear as the following:
![css-fix](https://github.com/perse-foni/WL_Delivery_Service/assets/4652993/8c2b61d3-6354-45c5-a80b-a3c0ff546deb)